### ISSUE TITLE
NTR: scromiting

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/human-phenotype-ontology#5098]
**Preferred term label:**
scromiting

**Definition (free text, please give PubMed ID)**
Screaming and vomiting at the same time.
https://www.therecoveryvillage.com/marijuana-addiction/faq/what-is-scromiting/#gref

not sure if we want to add this term, this article says it is not considered an official medical term.

**Parent term (use hpo.jax.org/app)**
HP_0002017 'Nausea and vomiting'

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
MONDO_0100094 Cannabinoid hyperemesis syndrome

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0001-5208-3432

See https://github.com/monarch-initiative/mondo/issues/1084

**Comment by pnrobinson** [comment]
This sounds pretty unpleasant. But in any case it is a "bundled" phenotype. HP_0002017 'Nausea and vomiting' is also bundled but is intended to be a grouping term for nausea-related phenotypes (is there a better label?).
So I would not add scromiting, I would think that hyperemesis and some other term for screaming would be better. Is there a good term to describe the screaming or do we need a new one?

**Comment by nicolevasilevsky** [comment]
is hyperemesis a thing by itself, or is it always hyperemesis gravidarum?

cc'ing @cmungall, as he originally mentioned the scromitting term on this ticket: monarch-initiative/mondo#1084

**Comment by pnrobinson** [comment]
hyperemesis just means "too much vomiting", and it is not always pregnancy related.

**Comment by pnrobinson** [comment]
I looked at the scromiting definition--when the pain becomes so bad that the person screams out. I would suggest we drop this now, this seems like a single case report and not necessarily a characteristic symptom.
